hw4/04-templates.js

```diff
@@ -51,7 +101,7 @@ app.get('/regions', (req, res) => {
   // reduce the output array in a resulting object that will feature the numbers of countries in each region
   // disregard empty data from the output array
 
-  let regions = ['Asia - 50', 'Europe - 53', 'Africa - 60'];
+  let regions = getList(req.url);
 
```

Comment:
Much better and simpler code than mine. It took me a long time to get done with this exercise.